src/main/java/com/adyen/httpclient/AdyenHttpClient.java

```diff
@@ -69,7 +69,9 @@
 
     private static final String CHARSET = "UTF-8";
     private static final String TERMINAL_CERTIFICATE_ALIAS = "TerminalCertificate";
+    private static final String JAVA_KEYSTORE = "JKS";
```

Comment:
I forgot this one :)
```suggestion
```

Reply:
> I forgot this one :)

my bad, i just applied your suggestion using Github UI instead of doing it in code.